jobserver/honeycomb.py

```diff
@@ -30,15 +37,19 @@ def format_honeycomb_timestamps(job):
     }
 
 
+def honeycomb_furl():
+    # TODO: make this configurable?
+    return furl(
+        "https://ui.honeycomb.io/bennett-institute-for-applied-data-science/environments/production/datasets/jobrunner"
+    )
```

Comment:
I'd just make this a module-level variable at this point, ie `BASE_URL = …`, then make the `furl(BASE_URL)` in the calling context.  I don't think we gain anything from the URL itself being in a utility method 🙂